src/github/issue-comment.test.ts

```diff
@@ -21,7 +21,7 @@ describe("Issue Comment Webhook", () => {
 		await Installation.create({
 			jiraHost,
 			clientKey: "client-key",
-			sharedSecret: "shared-secret"
+			encryptedSharedSecret: "shared-secret"
```

Comment:
So these tests are not directly related to the change.
But instead of mocking FF to true and/or false for all these test, I simply convert it to using the new col only, so that it is easier and closer to the final state we want.